dp-integrity-checker.nomad

```diff
@@ -0,0 +1,69 @@
+job "dp-integrity-checker" {
+  datacenters = ["eu-west-1"]
```

Comment:
The datacentre should now be `eu-west-2`. This should be updated in the generation tool. There is a temporary rewrite in CI so this will deploy, but all nomad plans will have to be updated eventually so we might as well get ahead with this one.

Reply:
Well spotted. Will update the generator templates too.

Reply:
Thank you 😄